src/htmlminifier.js

```diff
@@ -283,7 +283,7 @@ function cleanAttributeValue(tag, attrName, attrValue, options, attrs) {
     attrValue = trimWhitespace(attrValue);
     if (attrValue) {
       if (/;$/.test(attrValue) && !/&#?[0-9a-zA-Z]+;$/.test(attrValue)) {
-        attrValue = attrValue.replace(/\s*;$/, '');
+        attrValue = attrValue.replace(/\s*;$/, ';');
```

Comment:
This will preserve the trailing semi-colon regardless of `minifyCSS`, which is what I meant in https://github.com/kangax/html-minifier/pull/854#issuecomment-338923832

The trailing semi-colon is not needed for standard HTML, so if you need to preserve it, only do so for `minifyCSS:false`

Reply:
THis happens before minifyCss is executed, so minifyCss will throw out semicolons (unless configured otherwise) if minifyCss is true.
Should I add the check for minifyCSS === false anyways, to make it more explicit?

Reply:
I see what you mean - yes that is acceptable, as we'll now only trim whitespaces before trailing semi-colon, and `clean-css` takes care of the rest.

To make sure this works in the future, can you add a trailing semi-colon to the following test?
https://github.com/kangax/html-minifier/blob/9a45eb435a8a3ec453da52a752c0d4cf5c071b1d/tests/minifier.js#L2225